tests/screenshot.spec.ts

```diff
@@ -183,46 +211,7 @@ browserTest.describe('element screenshot', () => {
     await context.close();
   });
 
-  browserTest('should restore default viewport after fullPage screenshot', async ({ browser }) => {
```

Comment:
Same here, `captureBeyondViewport` still resizes the viewport under the hood, so we should make sure things are normal after capture.

Reply:
Ah, I didn't know -- thought these would become redundant. I'll resurrect them.